tests/test_pyxis.py

```diff
@@ -0,0 +1,426 @@
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import requests
+
+from http import HTTPStatus
+from copy import deepcopy
+from unittest.mock import call, patch, create_autospec, Mock
+
+from freshmaker import conf
+from freshmaker.pyxis import Pyxis
+
+from tests import helpers
+
+
+class TestQueryPyxis(helpers.FreshmakerTestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.koji_read_config_patcher = patch(
+            'koji.read_config', return_value={'server': 'http://locahost/'})
+        self.koji_read_config_patcher.start()
+
+        self.patcher = helpers.Patcher(
+            'freshmaker.pyxis.')
+
+        self.fake_server_url = 'https://pyxis.localhost/'
+        self.px = Pyxis(self.fake_server_url)
+        self.response = create_autospec(requests.Response)
+        self.response.status_code = HTTPStatus.OK
+        self.bad_requests_response = {
+            "detail": [
+                "Unable to parse the filter from URL.",
+                "Please verify the 'Field Name' in the RSQL Expression.",
+                "Please visit the following end-point for more details:",
+                "    /v1/docs/filtering-language"
+            ],
+            "status": 400,
+            "title": "Bad Request",
+            "type": "about:blank"
+        }
+
+        self.empty_response_page = {
+            "data": [],
+            "page": 0,
+            "page_size": 100,
+            "total": 0
+        }
+
+        self.indices = [
+            {
+                "_id": "1",
+                "created_by": "meteor",
+                "creation_date": "2020-01-01T09:32:31.692000+00:00",
+                "last_update_date": "2020-01-01T09:32:31.692000+00:00",
+                "last_updated_by": "meteor",
+                "ocp_version": "4.5",
+                "organization": "org",
+                "path": "path/to/registry:v4.5"
+            },
+            {
+                "_id": "2",
+                "created_by": "meteor",
+                "creation_date": "2020-01-01T09:32:38.486000+00:00",
+                "last_update_date": "2020-01-01T09:32:38.486000+00:00",
+                "last_updated_by": "meteor",
+                "ocp_version": "4.6",
+                "organization": "org",
+                "path": "path/to/registry:v4.6"
+            },
+            {
+                "_id": "2",
+                "created_by": "meteor",
+                "creation_date": "2020-01-01T09:32:38.486000+00:00",
+                "last_update_date": "2020-01-01T09:32:38.486000+00:00",
+                "last_updated_by": "meteor",
+                "ocp_version": "4.6",
+                "organization": "org",
+                "path": ""
+            }
+        ]
+
+        self.bundles = [
```

Comment:
Don't these bundles need `bundle_path_digest` set?

Reply:
Well since I was not using it in my tests I didn't add it there. If there will be tests that will need that info it can be added.